src/cli/onefuzz/templates/libfuzzer.py

```diff
@@ -17,6 +17,8 @@
 from . import JobHelper
 
 LIBFUZZER_MAGIC_STRING = b"ERROR: libFuzzer"
+LIBFUZZER_DOTNET_LOADER_PATH_LINUX = "/onefuzz/third-party/dotnet-fuzzing-linux/LibFuzzerDotnetLoader/LibFuzzerDotnetLoader.dll"
+LIBFUZZER_DOTNET_LOADER_PATH_WINDOWS = "/onefuzz/third-party/dotnet-fuzzing-windows/LibFuzzerDotnetLoader/LibFuzzerDotnetLoader.dll"
```

Comment:
Yes, this DLL is cross-platform. However, `libfuzzer-dotnet` needs a platform-specific wrapper to invoke the managed code under test as a child process. As a result, we (as of this PR) will deploy two copies of the DLL on VMs, but accompanied by a bunch of other platform native files to support the fuzzing task. The `dotnet_coverage` and `_crash_report` tasks just happen to only want/need the underlying DLL, because we're invoking its `main()` directly.

Reply:
can you please add this as a comment it will be useful to know when maintaining the code